packages/config-cli/src/cli/screenshot/screenshot.ts

```diff
@@ -58,85 +60,91 @@ export class CommandScreenShot extends CommandLineAction {
-    for (const test of TestTiles) {
-      const page = await chrome.newPage();
+export async function takeScreenshots(
+  host: string,
+  tag: string,
+  tiles: string,
+  chrome: Browser,
+  logger: LogType,
+): Promise<void> {
+  const TestTiles = await fsa.readJson<TileTestSchema[]>(tiles);
+  for (const test of TestTiles) {
+    const page = await chrome.newPage();
 
-      const tileSetId = await this.getTileSetId(test.tileSet, tag);
-      const styleId = await this.getStyleId(test.style, tag);
+    const tileSetId = await getTileSetId(test.tileSet, tag);
+    const styleId = await getStyleId(test.style, tag);
 
-      const searchParam = new URLSearchParams();
-      searchParam.set('p', test.tileMatrix);
-      searchParam.set('i', tileSetId);
-      if (styleId) searchParam.set('s', styleId);
+    const searchParam = new URLSearchParams();
+    searchParam.set('p', test.tileMatrix);
+    searchParam.set('i', tileSetId);
+    if (styleId) searchParam.set('s', styleId);
 
-      const loc = `@${test.location.lat},${test.location.lng},z${test.location.z}`;
-      const fileName = '.artifacts/visual-snapshots/' + host + '_' + test.name + '.png';
+    const loc = `@${test.location.lat},${test.location.lng},z${test.location.z}`;
+    const fileName = '.artifacts/visual-snapshots/' + host + '_' + test.name + '.png';
 
-      await mkdir(`.artifacts/visual-snapshots/`, { recursive: true });
+    await mkdir(`.artifacts/visual-snapshots/`, { recursive: true });
 
-      const url = `https://${host}/?${searchParam.toString()}&debug=true&debug.screenshot=true#${loc}`;
+    let url = `${host}/?${searchParam.toString()}&debug=true&debug.screenshot=true#${loc}`;
+    if (!url.startsWith('http')) url = `https"//${url}`;
 
-      logger.info({ url, expected: fileName }, 'Page:Load');
+    logger.info({ url, expected: fileName }, 'Page:Load');
 
-      await page.goto(url);
+    await page.goto(url);
 
-      try {
-        await page.waitForSelector('div#map-loaded', { state: 'attached' });
-        await page.waitForTimeout(1000);
-        await page.waitForLoadState('networkidle');
-        await page.screenshot({ path: fileName });
-      } catch (e) {
-        await page.screenshot({ path: fileName });
-        throw e;
-      }
-      logger.info({ url, expected: fileName }, 'Page:Load:Done');
-      await page.close();
+    try {
+      await page.waitForSelector('div#map-loaded', { state: 'attached' });
+      await page.waitForTimeout(1000);
+      await page.waitForLoadState('networkidle');
+      await page.screenshot({ path: fileName });
+    } catch (e) {
+      await page.screenshot({ path: fileName });
+      throw e;
     }
+    logger.info({ url, expected: fileName }, 'Page:Load:Done');
+    await page.close();
   }
+}
 
-  async getTileSetId(tileSetId: string, tag: string): Promise<string> {
-    if (tag === 'production') return tileSetId;
+async function getTileSetId(tileSetId: string, tag: string): Promise<string> {
```

Comment:
I think we can remove these now.

if we are pointing at a bundled config there is not `@tag` to worry about any more!

Reply:
Pr tag removed. We can just point a lambda function url as host to take screenshot for `linz/basemaps-config`.